physics/rotating_body.hpp

```diff
@@ -0,0 +1,84 @@
+﻿
+// The files containing the tree of child classes of |Body| must be included in
+// the order of inheritance to avoid circular dependencies.  This class will end
+// up being reincluded as part of the implementation of its parent.
+#ifndef PRINCIPIA_PHYSICS_MASSIVE_BODY_HPP_
+#include "physics/massive_body.hpp"
+#else
+#ifndef PRINCIPIA_PHYSICS_ROTATING_BODY_HPP_
+#define PRINCIPIA_PHYSICS_ROTATING_BODY_HPP_
+
+#include <optional.hpp>
+
+#include <vector>
+
+#include "geometry/grassmann.hpp"
+#include "quantities/named_quantities.hpp"
+#include "quantities/quantities.hpp"
+
+namespace principia {
+
+using geometry::AngularVelocity;
+using geometry::Instant;
+using geometry::Vector;
+using quantities::Angle;
+using quantities::AngularFrequency;
+
+namespace physics {
+
+template<typename Frame>
+class RotatingBody : public MassiveBody {
+  static_assert(Frame::is_inertial, "Frame must be inertial");
+
+ public:
+  class Parameters {
+   public:
+    Parameters(Angle const& reference_angle,
```

Comment:
Some documentation (describing the geometry or at least referring to the IAU WGCC report) would be welcome.